themes/bootstrap3/less/components/search.less

```diff
@@ -428,6 +428,9 @@ body.rtl {
   margin-left: 1rem;
   margin-right: 1rem;
   padding: 0;
+  .combined-list span.more-advanced-separator {
```

Comment:
In reviewing your latest class changes, I was reminded of this -- does this class no longer exist? Should we just remove the style changes, or am I overlooking something?

Reply:
No, you are right.  Fixed!